fs/fspath/path.go

```diff
@@ -36,6 +37,10 @@ func CheckConfigName(configName string) error {
 	if !configNameMatcher.MatchString(configName) {
 		return errInvalidCharacters
 	}
+	// Reject configName, if it starts with -, complicates usage. (#4261)
+	if strings.HasPrefix(configName, "-") {
```

Comment:
Can you add a test for this here

https://github.com/rclone/rclone/blob/17d5a724164bdbf22ca3118c38fca39dddaebd9d/fs/fspath/path_test.go#L12